contracts/EthZetaMock.sol

```diff
@@ -6,7 +6,7 @@ import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "@openzeppelin/contracts/utils/Context.sol";
 
 contract ZetaEthMock is ERC20("Zeta", "ZETA") {
-    constructor(address creator, uint256 initialSupply) {
-        _mint(creator, initialSupply * (10 ** uint256(decimals())));
+    constructor(uint256 initialSupply) {
```

Comment:
why this change? we change it to support creator in the protocol version because is easy to generate fancy addresses (ownership belongs to the factory)

Reply:
reverted https://github.com/zeta-chain/toolkit/pull/10/commits/e35a1ea3f054223087b4b63d51a9881703a3d71b